site/content/zh/latest/tasks/quickstart.md

```diff
@@ -0,0 +1,117 @@
+---
+title: "快速开始"
+weight: 1
+description: 只需几个简单的步骤即可开始使用 Envoy Gateway。
+---
+
+本指南将帮助您通过几个简单的步骤开始使用 Envoy Gateway。
+
+## 前置条件 {#prerequisites}
+
+一个 Kubernetes 集群。
+
+**注意：** 请参考[兼容性表格](../install/matrix)来查看所支持的 Kubernetes 版本。
+
+**注意：** 如果您的 Kubernetes 集群没有负载均衡器实现，我们建议安装一个
+，以便 Gateway 资源能够关联一个地址。我们推荐使用 [MetalLB](https://metallb.universe.tf/installation/)。
+
+## 安装 {#installation}
+
+安装 Gateway API CRD 和 Envoy Gateway：
+
+```shell
+helm install eg oci://docker.io/envoyproxy/gateway-helm --version v0.0.0-latest -n envoy-gateway-system --create-namespace
+```
+
+等待 Envoy Gateway 至可用后：
+
+```shell
+kubectl wait --timeout=5m -n envoy-gateway-system deployment/envoy-gateway --for=condition=Available
+```
+
+安装 GatewayClass，Gateway，HTTPRoute 和示例应用：
+
+```shell
+kubectl apply -f https://github.com/envoyproxy/gateway/releases/download/latest/quickstart.yaml -n default
+```
+
+**注意：**[`quickstart.yaml`] 定义了 Envoy Gateway 将侦听其全局可路由 IP 地址上端口
+80 上的流量，以便轻松使用浏览器测试 Envoy Gateway。当 Envoy Gateway 看到它的侦听器使用特权端口（<1024），
+它将在内部映射到非特权端口，因此 Envoy Gateway 不需要额外的特权。
+了解此映射很重要，当您调试时您可能需要将其考虑在内。
+
+[`quickstart.yaml`]: https://github.com/envoyproxy/gateway/releases/download/latest/quickstart.yaml
+
+## 测试配置 {#testing-the-configuration}
```

Comment:
can you do the same changes as in https://github.com/envoyproxy/gateway/pull/3208, thanks

Reply:
> can you do the same changes as in #3208, thanks

![image](https://github.com/envoyproxy/gateway/assets/34599468/ee2134a5-b231-4df0-8bd0-92d2d000b735)
Ok, finished. The css of shell syntax looks doesn't work just like https://github.com/envoyproxy/gateway/pull/3208#issuecomment-2058979807.

Reply:
This is okay, the css stylesheet will be added in the future.